index.html

```diff
@@ -77,40 +81,39 @@ <h2>Why?</h2>
         </p>
       </section>
 
+      <hr>
 
-      <section>
+      <section class="team">
         <h2>Meet the team</h2>
-        <ul>
-          <li>
+        <div class="CFO">
             <img src="https://placebear.com/150/150" alt="profile for kari kiwi" />
             <span>Kari Kiwi</span>
             <span>CFO</span>
-          </li>
-          <li>
+        </div >
+        <div class="CMO">
```

Comment:
Instead of using classes on these divs, this is a good opportunity to use an id instead.